container-images/ramalama/latest-rocm/Containerfile

```diff
@@ -0,0 +1,54 @@
+FROM quay.io/ramalama/ramalama:latest
+
+RUN /usr/bin/python3 --version
+
+ARG ROCM_VERSION=6.2.2
+ARG AMDGPU_VERSION=6.2.2
+
+RUN bash -c 'cat > /etc/yum.repos.d/amdgpu.repo <<EOF
```

Comment:
Lets create these files on the host system and copy them in rather then attempting to use <<EOF which is not supported on older versions of podman/buildah and maybe Docker.